docs/ref-info/ref-token-refresh.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Token Refresh
+description: Information for publishers about refreshing the UID2 token.
+hide_table_of_contents: false
+sidebar_position: 06
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+
+# Token Refresh
+
+When a publisher sends a user's <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> (email address or phone number) to the UID2 Operator, whether via one of the UID2 SDKs or the [POST /token/generate](../endpoints/post-token-generate.md) endpoint, the UID2 Operator returns a <a href="glossary-uid#gl-uid2-token">UID2 token</a>. The token is an opaque alphanumeric string, and is pseudonymous&#8212;this means that different instances of activity, on browsers, phones, and CTV, can be matched to the same pseudonymous value without compromising the privacy of the individual. The token cannot be reverse engineered to arrive at the original email address or phone number.
```

Comment:
_on browsers, phone, tablet and CTV devices?_
or 
_on browsers, devices and CTV?_

Both sound a bit weird so maybe leaving as above is fine...

Reply:
Updated to:
on browsers, CTV, and electronic devices such as phone and tablets,